### ISSUE TITLE
Accessibility in Db2 Developer Extension

### ISSUE THREAD

**Issue by Marty850** [IBM/db2forzosdeveloperextension-about#91]
Some of my colleagues would like to use screen readers like JAWS to work with the Db2 Developer Extension. Unfortunately, the Developer Extension doesn't work good with screen readers.

E.g. if you want to run a sql script, the focus is not set to a connection selection window. People that can't see don't use the mouse so it's very important for them that the focus is set to the next window and everything is selectable via keyboard.

Another example is when you want to read the result of a query, the screen reader is only able to read the headlines of the output. As workaround my colleague exports the result to csv and reads it in MS Excel. It would be better if the screen reader gets the results and you can navigate through with the arrow keys.

Microsoft does a lot for accessibility in VS Code (https://code.visualstudio.com/docs/editor/accessibility) it would be great if the Db2 Developer Extension would exploit those capabilities as well.

In our company it's very important for us to provide tools that are accessible to all users.

**Comment by katsoohoo** [comment]
Hi @Marty850,

Thank you for bringing up this accessibility issue. We will look into improving the experience with screen readers and see if we can leverage VS Code's capabilities.
